specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/2017-04-18/cognitiveservices.json

```diff
@@ -623,10 +623,11 @@
       "description": "Required. Indicates the type of cognitive service account.",
       "enum": [
         "Academic",
-        "Bing.Autosuggest",
-        "Bing.Search",
+        "Bing.Autosuggest.v7",
+        "Bing.CustomSearch",
+        "Bing.Search.v7",
         "Bing.Speech",
-        "Bing.SpellCheck",
+        "Bing.SpellCheck.v7",
```

Comment:
What happens if you pass the service the old string? If it doesn't spit back an error, we should really have these Swaggers continue to have the old values.

Reply:
No error. But those "old" APIs had been deprecated and we want to keep this list up to date.

Reply:
Aha, keeping it up to date is admirable! However, deleting the old entries can break other SDKs. Particularly, I'm pretty certain this would break the Go and Ruby SDKs. It drives home my point though, it seems like there should really be whole new API Versions with the updated versions of SpellCheck, etc.